src/dataregistry/registrar.py

```diff
@@ -170,9 +183,16 @@ def _handle_data(self, relative_path, old_location, verbose):
 
         return dataset_organization, num_files, total_size
 
-    _MAX_CONFIG = 10000
```

Comment:
Suggest keeping this line, except change `_MAX_CONFIG` to `_DEFAULT_MAX_CONFIG`.
then for register_execution change default value of `max_config_length` to `_DEFAULT_MAX_CONFIG`

Reply:
Done